docs/developer-guide/views-examples.md

```diff
@@ -0,0 +1,120 @@
+---
+description: Learn about the most used custom view examples of Lite XL.
+---
+
+## Toolbar
+
+### Tutorial
+
+```lua
+-- mod-version:3 -- lite-xl 2.1
+```
+
+This is required to specify Lite XL version compatibility.
+
+The instructions that begin with `require` add necessary libraries.
+
+```lua
+local toolbar = common.merge({
+  -- plugin configuration options go here
+}, config.plugins.build)
```

Comment:
why is it merging `config.plugins.build`? Can you give a more substantial example, like a config option and optionally illustrate the settings GUI API here?

Reply:
I think the settings GUI API should be documented in a separate document, because It could be used by any custom View.
Also, the reference to `build` was an error on my part.

Reply:
I'm working on some config options.